CRM/Event/Form/EventFees.php

```diff
@@ -414,7 +414,7 @@ public static function buildQuickForm(&$form) {
         CRM_Core_Payment_Form::buildPaymentForm($form, $form->_paymentProcessor, FALSE, TRUE);
       }
       elseif (!$form->_mode) {
-        $form->addElement('checkbox', 'record_contribution', ts('Record Payment?'), NULL,
+        $form->addElement('checkbox', 'record_contribution', ts('Record Invoice?'), NULL,
```

Comment:
Can you point me to the spec on this? It's an odd phraseology that seems incorrect.